setup.py

```diff
@@ -60,6 +60,7 @@
     'discord.commands',
     'discord.ext.commands',
     'discord.ext.tasks',
+    'discord.ext.menus'
```

Comment:
```suggestion
    'discord.ext.menus',
```
Add magic trailing comma to allow for less lines changed when future `ext` modules are added